zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -4195,6 +4200,14 @@ components:
         value:
           type: string
           description: The value of the variable.
+    VariableUserTaskFilterRequest:
```

Comment:
```suggestion
    UserTaskVariableFilterRequest:
```

Reply:
`UserTaskVariableFilterRequest` already exists.
Do you have maybe another idea? I had the same thoughts over here

Reply:
Oh, I wasn't aware, thanks for that hint 😄 
In that case, just leave it as is, your naming is fine then 👍